app/components/agenda/agenda-header/agenda-actions.js

```diff
@@ -333,6 +333,42 @@ export default class AgendaAgendaHeaderAgendaActions extends Component {
     this.args.didApproveAgendaitems();
   }
 
+  @action
+  async markDecisionsForSigning() {
+    const agenda = this.args.currentAgenda;
+    const agendaitems = await agenda.agendaitems;
```

Comment:
see other comment, we can use meeting for query instead of agenda and the agendaitems.
Regardless: `agenda.agendaitems` is something we should avoid for cache / stale data / concurrency reasons.
We normally do a query to `agendaitems` with a filter on agenda id and a page size (or queryAll would work)

This info is not readily available so I might add it to the agenda.js model with a warning on it. You could not have known this. We only use it in older code and should really be refactored to more specific queries tbh.
Also because normally there is a page size limit on relations. you would only get 20 results from a list this way unless we write an adapter for it (used to be the case, need to verify if that is still a thing)